collectors/ebpf.plugin/ebpf_functions.c

```diff
@@ -1070,19 +1587,24 @@ void *ebpf_function_thread(void *ptr)
     (void)ptr;
 
     bool ebpf_function_plugin_exit = false;
-    struct functions_evloop_globals *wg = functions_evloop_init(1,
+    struct functions_evloop_globals *wg = functions_evloop_init(2,
                                                                 "EBPF",
```

Comment:
Hello @ktsaou ,

I am pinging you here, because I had a chat with @stelfrag about a code we already merged and we are using here in this PR. As you can see we are passing the variable `ebpf_function_plugin_exit` for `functions_evloop_init` and this variable is used to control plugin exit according the variable it is [assigned](https://github.com/netdata/netdata/blob/060f3ff7bf5b91899280ceaefaddf7ff4016a1f6/libnetdata/functions_evloop/functions_evloop.c#L180). 

eBPF plugin has different threads and I understand that instead to pass this variable as argument here, in the ideal scenario we should pass it as argument for the function `functions_evloop_add_function`. With this change, it would be possible to cancel either the plugin or only a thread. If somehow @stelfrag and I are missing something, please, let us know. On the other hand, if you accept the suggestion, I was thinking we can work this in another PR to simplify my reviewers.

Best regards!